stories/Atom/Images/Imagecaptioncredit/Imagecaption.jsx

```diff
@@ -26,10 +26,10 @@ export const Images = ({
 }) => {
   useEffect(() => {
     viewport('.image-section__cart');
-    viewport('.image-section__description');
+    viewport('.image-section .image__caption');
   }, []);
   return (
-    <div className={['grid-x', 'image-section', `${size_options3[`${args.size}`]}`].join(' ')}>
+    <figure className={['grid-x', 'image-section', `${size_options3[`${args.size}`]}`].join(' ')}>
```

Comment:
@Shamimkhan321 Kindly use `cls` to add/toggle classes as this adds a whitespace in class attrib. Similarly in the below line.